manifests/art.yaml

```diff
@@ -10,5 +10,5 @@ updates:
       replace: 'olm.skipRange: ">=4.3.0-0 <{FULL_VER}"'
   - file: "sriov-network-operator.package.yaml"
     update_list:
-    - search: "currentCSV: sriov-network-operator.v4.4.0"
+    - search: "currentCSV: sriov-network-operator.v4.5.0"
```

Comment:
Perhaps change this to `currentCSV: ptp-operator.v{MAJOR}.{MINOR}.0` so that we don't need to update the version for every release.

Reply:
Fixed.